common/cpp/s3_rate_limiter.cpp

```diff
@@ -44,7 +44,7 @@ class S3RateLimiter::SimpleSpinLock {
             spin_count++;
             if (spin_count >= MAX_SPIN_COUNT) {
                 LOG(WARNING) << "Warning: Excessive spinning detected while acquiring lock. Spin "
-                                "count: ";
+                                "count: " << spin_count;
```

Comment:
do not modify this line?

Reply:
fix it with a separate PR

Reply:
here， in config.cpp


// clang-format off
#ifdef BE_TEST
// test s3
DEFINE_String(test_s3_resource, "resource");
DEFINE_String(test_s3_ak, "ak");
DEFINE_String(test_s3_sk, "sk");
DEFINE_String(test_s3_endpoint, "endpoint");
DEFINE_String(test_s3_region, "region");
DEFINE_String(test_s3_bucket, "bucket");
DEFINE_String(test_s3_prefix, "prefix");
#endif
// clang-format on